semgrep-core/tests/csharp/metavar_call.cs

```diff
@@ -1,4 +1,4 @@
-public static void Test
+public static void Test()
```

Comment:
Does `public` work outside of a class context?

Reply:
OK I just put everything inside a class so there is no doubt that it is valid C#.